02week/bio/resume.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link href="/static/fontawesome/fontawesome-all.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.10/css/all.css" integrity="sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/resumestyle.css" />
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet" type="text/css">
+    <meta charset="utf-8">
+    <title>Darwin Risser Resume</title>
+  </head>
+
+  <header role=”banner”>
+    Darwin Risser<br />
+    <a href="risser.darwin@gmail.com" target="_blank">risser.darwin@gmail.com   <i class="fa fa-address-book"></i></a><br />
```

Comment:
double check how to link to emails, it works a little differently than a normal href

Reply:
Thanks Shelby, got it working now!